### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by AllKinds**

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -1,24 +1,14 @@
...
-        persistent: false,
-    },
     server: {
         port: +(process.env.PORT || 8080),
-        name: 'authentication',
+        name: 'authentication-service',
+        endpoint: process.env.SERVER_ENDPOINT || 'http://localhost:3000/auth',
     },
     cors: {
         allowedOrigins: process.env.ALLOWED_ORIGINS ? process.env.ALLOWED_ORIGINS.split(',') : ['http://localhost:4200'],
```
Comment (omerzamir) [inline src/config.ts]: change default origin to http://localhost/

File: src/config.ts [hunk 2]
```diff
@@ -1,24 +1,14 @@
...
-        port: +(process.env.RMQ_LOGGER_PORT || 15672),
-        password: process.env.RMQ_LOGGER_PASS || 'guest',
-        username: process.env.RMQ_LOGGER_USER || 'guest',
-        persistent: false,
-    },
     server: {
         port: +(process.env.PORT || 8080),
-        name: 'authentication',
+        name: 'authentication-service',
+        endpoint: process.env.SERVER_ENDPOINT || 'http://localhost:3000/auth',
```
Comment (omerzamir) [inline src/config.ts]: change default endpoint to http://localhost:3000/auth

File: package.json [hunk 3]
```diff
@@ -38,12 +41,12 @@
     "@grpc/proto-loader": "^0.4.0",
     "body-parser": "^1.18.3",
     "express": "^4.16.4",
-    "grpc": "^1.19.0",
+    "grpc": "^1.20.3",
     "helmet": "^3.14.0",
     "jayson": "^2.1.1",
```
Comment (omerzamir) [inline package.json]: remove jayson- unnecessary

**Comment by omerzamir** [comment]
Also, you have a conflict to solve.

File: src/config.ts [hunk 4]
```diff
@@ -28,6 +18,7 @@ export const config = {
         required: true,
         token: process.env.TOKEN || 'kd-token',
         secret: process.env.SECRET_KEY || 'pandora@drive', // TODO: Don't use static value in production! remove from source control!
+        daysExpires: +(process.env.TOKEN_DAYS_EXPIRES || 30),
         saml: {
             entryPoint: process.env.SAML_ENTRY_POINT || 'http://localhost:8080/simplesaml/saml2/idp/SSOService.php',
             issuer: process.env.SAML_ISSUER || 'http://localhost:3000/metadata.xml',
```
Comment (omerzamir) [inline src/config.ts]: issuer should also be changed to `http://localhost:8080/auth/metadata.xml`, missed it too...

File: tsconfig.json [hunk 5]
```diff
@@ -13,6 +13,7 @@
     "src/**/*.ts"
   ],
   "exclude": [
+    "src/**/*.spec.ts",
```
Comment (Shahar-Y) [inline tsconfig.json]: is it necessary?

Comment (AllKinds) [reply tsconfig.json]: Not really. But the tests are running with ts-node, so you don't realy need the spec.js files anyway.

**Pull request merged**